librz/analysis/fcn.c

```diff
@@ -1765,14 +1740,13 @@ RZ_API RzAnalysisFunction *rz_analysis_get_function_byname(RzAnalysis *a, const
 
 /* rename RzAnalysisFunctionBB.add() */
 RZ_API bool rz_analysis_fcn_add_bb(RzAnalysis *a, RzAnalysisFunction *fcn, ut64 addr, ut64 size, ut64 jump, ut64 fail, RZ_BORROW RzAnalysisDiff *diff) {
-	D eprintf("Add bb\n");
 	if (size == 0) {
-		eprintf("Warning: empty basic block at 0x%08" PFMT64x " is not allowed.\n", addr);
+		RZ_LOG_ERROR("empty basic block at 0x%08" PFMT64x " is not allowed.\n", addr);
```

Comment:
Capitalize the first letter of the error message sentence to be consistent with all other places.